src/test/resources/features/extractMetadata/extract.feature

```diff
@@ -0,0 +1,69 @@
+Feature: Extract Metadata
+
+  In order to extract metadata from a file
+  As a valid Converter customer
+
+  Background: Sets authentication
+    Given I set valid authentication headers
+  
+  @functional
+  Scenario: Verify that is possible to extract metadata from an file with "exportFormat" field as TXT
```

Comment:
Please consider improving this title to something like: `Verify that is possible to extract metadata from a file using TXT export format`